src/components/investors/CTAComponent.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { CTADivider } from '../shared/dividers/CTADivider';
+import { openInNewTab } from '../../pages';
+
+export const CTAComponent = () => {
```

Comment:
same for this, I would use a unique name for the InvestorsCTAComponent
```suggestion
export const InvestorsCTAComponent = () => {
```